api/proto/metrics/metrics.proto

```diff
@@ -468,6 +468,17 @@ message ZMetricVolume {
   uint64 freeBytes = 9;
 }
 
+// Metrics for a snapshot
+message ZMetricSnapshot {
+  string uuid = 1;
+  string display_name = 2;         // User-friendly name
+  uint64 used_space = 3;          // In byte
+  uint64 referenced = 4;          // In byte
+  double compressratio = 5;       // Compression ratio
+  uint64 snapshot_size = 6;       // In byte
+  uint64 logicalreferenced = 7;   // In byte
```

Comment:
Do some of these counters indicate how much space is used uniquely by this particular snapshot i.e., space which would be freed up if the snapshot is deleted? It would be good to document that in comments.

Reply:
I added a description